src/main/java/Exercise04.java

```diff
@@ -0,0 +1,23 @@
+public class Exercise04 {
+    public String AnalyzeNumber(int number) {
+        StringBuilder result=new StringBuilder();
+        int count=0;
```

Comment:
Biến count có thể được khai báo ngay trong vòng for

Reply:
done